ruby/lib/minitest/queue.rb

```diff
@@ -102,6 +102,21 @@ def mark_as_flaked!
     end
   end
 
+  module TimeDetails
+    def start_timestamp
+      @start_timestamp
+    end
+
+    def end_timestamp
```

Comment:
[nitpick] I usually try to avoid `end` as it's a reserved word in Ruby. So I think I would prefer `finish` here.